### ISSUE TITLE
Invalid links in documentation

### ISSUE THREAD

**Issue by modelica-trac-importer** [tbeu/Modelica_Synchronous#4]
**Reported by dietmarw on 30 Oct 2013 13:25 UTC**
The OpenModelica documentation generator also checks for dead links. Here the result (taken from https://build.openmodelica.org/Documentation/FindFiles.log:
```
Modelica_Synchronous.ClockSignals.Clocks.PeriodicRealClock.html: Not found: Modelica_Synchronous.Clocks.PeriodicExactClock.html
Modelica_Synchronous.RealSignals.Periodic.MovingAverage.html: Not found: Modelica_Synchronous.Periodic.FIRbyCoefficients.html
Modelica_Synchronous.RealSignals.Sampler.Utilities.Internal.random.html: Not found: Modelica.UsersGuide.ModelicaLicense.html
Modelica_Synchronous.Types.Resolution.html: Not found: Modelica_Synchronous.Clocks.PeriodicExactClock.html
Modelica_Synchronous.WorkInProgress.Incubate.FIRbyWindow.html: Not found: Modelica_Synchronous.Periodic.FIRbyCoefficients.html
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1341

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 24 Jan 2016 13:27 UTC**
Fixed in 8b01c0d980a6035a08e8dbb679359ee6c70924e9.

**Comment by modelica-trac-importer** [comment]
**Changelog modified by otter on 24 Jan 2016 13:27 UTC**
Invalid links fixed in documentation layers
